src/service/overseas.entities.service.ts

```diff
@@ -7,13 +7,10 @@ import { Transactionkey, OverseasEntityKey } from "../model/data.types.model";
 import { makeApiCallWithRetry } from "./retry.handler.service";
 import { ApplicationData } from "../model/application.model";
 
-// TODO find all calls to this function and remove the isSaveAndResumeFeatureActive
-// parameter when registration and update save and resume flags are removed
 export const createOverseasEntity = async (
   req: Request,
   session: Session,
   transactionId: string,
```

Comment:
V minor, but comma at end of final parameter can go.

Reply:
done